app/routes.py

```diff
@@ -1,2 +1,53 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify
+
+class Planet:
+    def __init__(self, id, name, description, color):
+        self.id = id
+        self.name = name 
+        self.description = description
+        self.color = color
+    
+    def return_planets(self):
+        return {
+            "id": self.id,
+            "name": self.name,
+            "description": self.description,
+            "color": self.color  
+        }
+        
+
+planets = [
+    Planet(1,"Mercury", "hot", "brown"), 
+    Planet(2, "Venus", "pretty", "redish brown"),
+    Planet(3, "Earth", "wet", "greenis blue"),
+    Planet(6, "Saturn", "rings", "purple and yellow" )
+
+]
+    
+
+solar_systems_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+@solar_systems_bp.route("", methods=["GET"])
+def get_planet_json():
+    planet_response = []
+    for planet in planets:
+        planet_response.append(
+            planet.return_planets()
+        )
+    return jsonify(planet_response)
+
+@solar_systems_bp.route("/<planet_id>", methods=["GET"])
+def get_each_planet_with_id(planet_id):
+    planet_id = int(planet_id)
+    
+
+    for planet in planets:
+        if planet.id == planet_id:
+            return jsonify(planet.return_planets())
+
+    return {
+        "error": (f"ID {planet_id} not exists"),
+        "status": "404"
+    },404 
```

Comment:
Great work handling not found planets! Since the response code will be returned as 404, it is not necessary to include this in the json